src/DetailsView/components/assessment-test-view.tsx

```diff
@@ -124,7 +75,22 @@ export const AssessmentTestView = NamedFC<AssessmentTestViewProps>(
                     test={assessmentNavState.selectedTestType}
                 />
                 <TargetChangeDialog deps={deps} prevTab={prevTarget} newTab={currentTarget} />
-                {mainContent}
+                {isGettingStartedSelected ? (
```

Comment:
Usually don't like having conditionals in the component html itself but this reads quite nicely.